woocommerce/woocommerce
# Add marketplace theme cards

### Changes proposed in this Pull Request:

This PR is part of the WooCommerce Marketplace project.



This PR updates ProductCards to support themes. It also introduces ProductTypes, and fixes some labels in the Themes component.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Rebuild assets
2. Login as a admin
3. Open the [in-app marketplace](http://localhost:8888/wp-admin/admin.php?page=wc-admin&path=%2Fextensions&tab=themes&category=themes) (click this link to get the correct path to display themes while API is still being finalised)
4. Click on the _Themes_ tab
5. See the new theme cards and confirm that they match the design



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Update ProductCard to support themes

#### Comment 

</details>

### Screenshots
![Screenshot 2023-09-18 at 07 59 27](https://github.com/woocommerce/woocommerce/assets/1199991/5baff627-a4de-4415-b425-407d715e4c66)
![Screenshot 2023-09-18 at 07 59 14](https://github.com/woocommerce/woocommerce/assets/1199991/ff552449-04be-4567-94cd-5d83f4fa9ce1)
